### PR TITLE
First draft of IPFS Weekly #40

### PR DESCRIPTION
**Pull request by jennwrites**
Looking for feedback and suggestions on this draft so that we can send it out via TinyLetter (and post on the blog) Tuesday, April 30th. 🐢

Thanks!

### CODE REVIEW

**Comment by agentofuser** [comment]
Oh wow, I was shocked to see my humble Interplanetary Gatsby blog linked there 😅 It's still under development though and I'd like to spruce it up a bit before having it go out to everyone, would that be ok? I was going to announce the blog to the world after finishing an upcoming post on [ipfs-deploy](https://github.com/agentofuser/ipfs-deploy). Anyway, thank you so much for noticing it 😊

File: published/040-2019-apr-30.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+[IPFS Camp](https://blog.ipfs.io/72-ann-ipfs-camp/) is taking place June 27th-30th in beautiful Barcelona. Meet the core developers and contributors to the IPFS project, community leaders, and builders on of the Distributed Web, but only if you register to attend. [Registration is open now!](https://camp.ipfs.io/)
+ 
+ 
+## Coming up in the Community
+*Did you know IPFS has a community forum at [discuss.ipfs.io](https://discuss.ipfs.io/)? Sign up to participate in discussions about coding, tutorials, see announcements and learn about upcoming community events.*
+
+
++ **2 May 2019:** [Munich IPFS User Group](https://www.meetup.com/de-DE/Munich-IPFS-User-Group/events/259762490/): [Pierre Krieger](https://twitter.com/tomaka17) from [parity](https://www.parity.io/) talks about Rust libp2p.
++ **2-3 May 2019:** [Uphill Conf](https://uphillconf.com/) – The inspiring frontend conference on top of beautiful mount Gurten, Bern Switzerland.
```
Comment (Stebalien) [inline published/040-2019-apr-30.md:49]: Could we get an announcement for an SF IPFS Meetup on May 9th?

Details:

* Where: 2169 Folsom Street, Apt M202
* When: May 9th, 6pm-9pm
* What: Come hack on IPFS.

**Comment by jennwrites** [comment]
@agentofuser It’s been a few days since your comment, do you still want me to hold off on sharing?

File: published/040-2019-apr-30.md (hunk 1 above)
Comment (jennwrites) [reply published/040-2019-apr-30.md:49]: @Stebalien Do you have a link we can share here?

**Comment by jennwrites** [comment]
IPFS Weekly #40 was published

On the [IPFS blog](https://blog.ipfs.io/85-ipfs-weekly-40/)
Sent to 2936 subscribers [via Tinyletter](https://tinyletter.com/ipfsnewsletter/letters/ipfs-weekly-40)

**Comment by agentofuser** [comment]
@renrutnnej hi, sorry, for some reason I did not get a notification for your message on github's website, and I just now saw the notification email. Thank you for waiting to share. Can I ping you when I have the announcement post ready? Thanks again!

**Comment by jennwrites** [comment]
@agentofuser No worries! The next issue goes out May 7 and after that May 14 so just let me know whenever you're ready. Thanks ☺️

**Pull request merged**